src/pages/patientView/PatientViewPage.tsx

```diff
@@ -47,6 +51,7 @@ interface IPatientViewState {
     cnaSegmentData: any;
     mutationData: any;
     hotspotsData?: IHotspotData;
+    cosmicData?: ICosmicData;
```

Comment:
how about moving this (and IHotspotData) definition into this file and importing it from here in the column formatters?